spec/src/main/asciidoc/configexamples.asciidoc

```diff
@@ -52,6 +52,14 @@ $> java -jar some.jar -Dacme.myprj.some.url=http://other.server/other/endpoint
 Note that this is only one example how to possibly configure your application.
 Another example is to register <<custom_configsources, Custom ConfigSources>> to e.g. pick up values from a database table, etc.
 
+If a config value is `,` separated string, this value can be automatically converted to an array with `\` as the escape character.
```

Comment:
> is *a* `,` separated string

Also, does this imply that if its not separated it should not be a `String[]` of size 1?

Reply:
yes, if the escape character is before the comma

Reply:
reworded "If a config value is `,` separated string, this value can be automatically converted to a multiple element array with `\` as the escape character."